libraries/anaconda3-runtime/context/pipe

```diff
@@ -2,10 +2,10 @@
 
 set -e
 
-CONDAENV=conda_environment
+ANACONDA_ENV=/home/algo/conda_environment
 ROOTDIR=/opt/algorithm
 
 . /opt/conda/etc/profile.d/conda.sh
-conda activate $CONDAENV
+conda activate ANACONDA_ENV
```

Comment:
Here too. You aren't interpreting the `ANACONDA_ENV` variable

Reply:
fixed in latest version